.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -34,15 +34,16 @@ public class OpenAIClient {
         public virtual ModerationClient GetModerationClient(string model);
         public virtual VectorStoreClient GetVectorStoreClient();
     }
-    public class OpenAIClientOptions : ClientPipelineOptions {
+    public class OpenAIClientOptions {
```

Comment:
The `export-api.ps1` script is behaving a bit unexpectedly; I'm guessing the rigid `--assembly-reference` paths in the script don't match your local environment and it's silently not picking up the dependencies for clean parity output.

Doesn't need to be fixed right now and the exported API is still very useful to see the new surface, but we can (a) see if we can get a clean baseline for you and (b) should see if we can make those assembly references more robust somehow.